src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -61,6 +61,9 @@ class ReflectionExtractor implements PropertyListExtractorInterface, PropertyTyp
      */
     private $arrayMutatorPrefixes;
 
+    private $arrayMutatorPrefixesFirst;
```

Comment:
We don't need this class property if we do some refacto on the existing $mutatorPrefixes and $arrayMutatorPrefixes ones. Basically, currently, $mutatorPrefixes includes the values of $arrayMutatorPrefixes. We could separate the values and join them when we need them.

Reply:
As these properties are marked `@internal`, doing the refactor is ok. I'm in favor for the refactor if it simplifies the code.

Reply:
the refacto should target master